src/monmove.cpp

```diff
@@ -382,83 +372,73 @@ void monster::move()
     }
 
     if( current_attitude == MATT_IGNORE ||
-        ( current_attitude == MATT_FOLLOW && plans.size() <= MONSTER_FOLLOW_DIST ) ) {
+        ( current_attitude == MATT_FOLLOW && rl_dist(pos(), goal) <= MONSTER_FOLLOW_DIST ) ) {
```

Comment:
Not wrong, but this changes the behavior a bit: old version was identical for both metric, in new one euclidean metric makes monsters follow more closely.